fastembed/text/onnx_embedding.py

```diff
@@ -16,6 +16,7 @@
         "sources": {
             "hf": "yashvardhan7/bge-base-en-onnx",
         },
+        "model_file": "model.onnx",
```

Comment:
The path can be configured. So you didn't have to move.

Reply:
Yikes. You moved it.

Reply:
but for most of the model , the path will remain as `onnx/model.onnx` .So it is necessary

Reply:
Conventionally, yes.